src/Dapplo.Microsoft.Extensions.Hosting.Wpf/IWpfBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Dapplo and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dapplo.Microsoft.Extensions.Hosting.Wpf
+{
+    /// <summary>
+    /// Interface used for configuring Wpf 
+    /// </summary>
+    public interface IWpfBuilder
```

Comment:
The name is not 100% clear, we don't build Wpf but a WPF Configuration.
IWpfConfigurationBuilder might be more suitable.